### PR TITLE
updated default env to use arbitrum nitro goerli

### PR DESCRIPTION
**Pull request by kautukkundan**
## What is this PR doing?
Defaults the network of extension and aggregator to arbitrum nitro goerli chain

## How can these changes be manually tested?
- clear existing postgres container
- build extension 
- import extension, create new wallet and send tx
- the tx should be sent on nitro

## Does this PR resolve or contribute to any issues?
#310 

## Checklist
- [x] I have manually tested these changes
- [x] Post a link to the PR in the group chat

## Guidelines
- If your PR is not ready, mark it as a draft
- The `resolve conversation` button is for reviewers, not authors
  - (But add a 'done' comment or similar)

### CODE REVIEW

File: extension/.env.example
Comment (kautukkundan) [inline extension/.env.example]: before merging @voltrevo should update the public aggregator running on this endpoint to use [this](https://github.com/web3well/bls-wallet/blob/new-deployment/contracts/networks/arbitrum-testnet-goerli.json) config

**Comment by voltrevo** [comment]
To reduce confusion, how about we use the name `arbitrum-goerli`/`Arbitrum Görli`? (E.g. aggregator url would be `https://arbitrum-goerli.blswallet.org`) This would be consistent with [chainlist](https://chainlist.org/chain/421613).

**Comment by kautukkundan** [comment]
I agree with you. Updated.

File: docs/local_development.md
Comment (voltrevo) [inline docs/local_development.md]:
```suggestion
By default the extension will connect to contracts already deployed on Arbitrum Nitro testnet and a public Aggregator running on https://arbitrum-goerli.blswallet.org/
```

File: docs/local_development.md
Comment (voltrevo) [inline docs/local_development.md]:
```suggestion
Then choose to target either a local Hardhat node or the Arbitrum Testnet. If you choose to run on Arbitrum Görli skip ahead until tests.
```

File: extension/.env.example
Comment (voltrevo) [inline extension/.env.example]:
```suggestion
AGGREGATOR_URL=https://arbitrum-goerli.blswallet.org/
```

**Comment by voltrevo** [comment]
Good to go when (if? 😅) https://github.com/web3well/bls-wallet/pull/333 is merged into here

**Review by voltrevo — APPROVED** [review]

**Pull request merged**
